### ISSUE TITLE
Flakey windows functional test - TestTelemetryWithStatsPolling

### ISSUE THREAD

**Issue by fenxiong** [aws/amazon-ecs-agent#1910]
### Summary

TestTelemetryWithStatsPolling failing intermittently on windows:
```
Error Trace:	functionaltests_test.go:642
        	            				functionaltests_windows_test.go:203
        	Error:      	Received unexpected error:
        	            	Timed out waiting for task 'STOPPED' to reach 
```

### Expected Behavior
Test passes

### Observed Behavior
Test failed

**Comment by yumex93** [comment]
This test can fail due to other reasons:
````
=== RUN   TestTelemetryWithStatsPolling
--- FAIL: TestTelemetryWithStatsPolling (277.48s)
	utils_unix.go:130: Created directory /var/log/functional-tests/ecs_integ_testdata591306224 to store test data in
	utils_unix.go:146: Launching agent with image: amazon/amazon-ecs-agent:latest
	utils_unix.go:233: Agent started as docker container: 456386580b792290a4fa4ce3dbefb71820e6752f11ba84c8042c146ceb154fb4
	utils.go:175: Found agent metadata: {Cluster:ecstest-telemetry-polling-25624858-1c03-4c3b-bed4-0bd2b24c992f ContainerInstanceArn:0xc420421050 Version:Amazon ECS Agent - v1.26.0 (ebac2200)}
	utils.go:196: Task definition: ecs-metrics-test-356c9a551fa37a7563b192ec719173eb:1
	utils.go:216: Started task: arn:aws:ecs:us-west-2:355690550782:task/a78d51c1-1dde-4223-8de5-7a4e48ba42a0
	functionaltests_test.go:627: 
			Error Trace:	functionaltests_test.go:627
			            				functionaltests_unix_test.go:385
			Error:      	Max difference between 20.833333333333336 and 14.512813329696655 allowed is 5, but difference was 6.32052000363668
			Test:       	TestTelemetryWithStatsPolling
````

**Comment by fierlion** [comment]
A fix for this was shipped in agent 1.26
